t/30-elembio-runstats.t

```diff
@@ -125,11 +125,14 @@ use_ok('npg_qc::elembio::run_stats');
                 is($lane->tags->{0}, 'NNNNNNNN',
                     'Tag 0 virtual barcode is correct');
             } else {
-                ok($lane->tags->{$tag_index} =~ /^[ATCG]{8}$/,
+                my ($is_control) = grep {$_ == $tag_index} (1,2,3,4);
+                my $expr = $is_control ? qr/^[ATCG]{8}\(CTRL\)$/ : qr/^[ATCG]{8}$/;
+                ok($lane->tags->{$tag_index} =~ $expr,
```

Comment:
You could use `Test::More::like` here if you wanted. You get slightly nicer output when the test fails.